.evergreen/run-orchestration.sh

```diff
@@ -101,21 +101,21 @@ URI=$(python -c 'import sys, json; j=json.load(open("tmp.json")); print(j["mongo
 echo 'MONGODB_URI: "'$URI'"' > mo-expansion.yml
 echo $URI > $DRIVERS_TOOLS/uri.txt
 echo "Cluster URI: $URI"
-
-if [ -z $MONGO_CRYPT_SHARED_DOWNLOAD_URL ]; then
-  echo "There is no crypt_shared library for distro='$DISTRO' and version='$MONGODB_VERSION'".
-else
-  echo "Downloading crypt_shared package from $MONGO_CRYPT_SHARED_DOWNLOAD_URL"
-  download_and_extract_crypt_shared "$MONGO_CRYPT_SHARED_DOWNLOAD_URL" "$EXTRACT" CRYPT_SHARED_LIB_PATH
-  echo "CRYPT_SHARED_LIB_PATH:" $CRYPT_SHARED_LIB_PATH
-  if [ -z $CRYPT_SHARED_LIB_PATH ]; then
-    echo "CRYPT_SHARED_LIB_PATH must be assigned, but wasn't" 1>&2 # write to stderr"
-    exit 1
-  fi
-cat <<EOT >> mo-expansion.yml
+if [ -z ${SKIP_CRYPT_SHARED+''} ]; then
```

Comment:
What is the purpose of `+''`?

Reply:
It is normally used when you want a default value for a variable expansion: ${SKIP_CRYPT_SHARED+'default value if variable is not defined'}. However, in this case we do not need it, so I removed it.

Reply:
Oh in that case you should use this style which is what we use elsewhere:
```
if [ -z "${SKIP_CRYPT_SHARED:-}" ]; then
```

Reply:
Please also add a comment that says something like:
```
# Define SKIP_CRYPT_SHARED=1 to skip downloading crypt_shared, this is useful 
# for platforms that have a server build but don't ship a corresponding crypt_shared
# release, like Amazon 2018.
```

Reply:
Done.